RHEL/7/input/xccdf/system/accounts/pam.xml

```diff
@@ -313,25 +313,25 @@ run of (<sub idref="var_password_pam_maxrepeat" /> + 1) or more identical charac
 To check the maximum value for consecutive repeating characters, run the following command:
 <pre>$ grep maxrepeat /etc/security/pwquality.conf</pre>
 Look for the value of the <tt>maxrepeat</tt> parameter. The DoD requirement is 2, which would appear as
```

Comment:
@shawndwells since `var_password_pam_maxclassrepeat` was changed to 4, shouldn't this equal 4 as well?